app/components/TaskList/index.tsx

```diff
@@ -263,14 +270,10 @@ const TaskList: React.FC<Props> = ({
           </TaskListSection>
         )}
         {/* Amendments & Other Revisions section */}
-        {useShowGrowthbookFeature("amendments") && mode === "view" && (
+        {useShowGrowthbookFeature("amendments") && mode !== "create" && (
           <ProjectRevisionChangeLogsTaskListSection
             projectRevisionId={id}
-            defaultExpandedState={[
-              getProjectRevisionChangeLogsPageRoute(id).pathname,
-              projectRevisionViewPagePathName,
-              projectRevisionCreatePagePathName,
-            ].includes(router.pathname)}
+            defaultExpandedState={true}
```

Comment:
Since we are not using `ProjectRevisionChangeLogsTaskListSection` anywhere else, I think we can safely remove the `defaultExpandedState` from this component and set the `isExpanded` state always true.